File: src/main/java/net/hectus/neobb/turn/default_game/block/TComposter.java
package net.hectus.neobb.turn.default_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.attributes.clazz.NatureClazz;
import net.hectus.neobb.turn.default_game.attributes.function.AttackFunction;
import org.bukkit.block.Block;

public class TComposter extends BlockTurn implements AttackFunction, NatureClazz {
    public TComposter(NeoPlayer player) { super(player); }
    public TComposter(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 2;
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/block/TComposter.kt
package net.hectus.neobb.modes.turn.default_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.NatureClazz
import net.hectus.neobb.modes.turn.default_game.attribute.function.AttackFunction
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class TComposter(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), AttackFunction, NatureClazz {
    override val cost: Int = 2
}
